WordPressAuthenticator/Signin/LoginPrologueViewController.swift

```diff
@@ -436,15 +437,6 @@ class LoginPrologueViewController: LoginViewController {
         navigationController?.pushViewController(toVC, animated: true)
     }
 
-    private func presentGetStartedView() {
-        guard let toVC = GetStartedViewController.instantiate(from: .getStarted) else {
-            DDLogError("Failed to navigate to GetStartedViewController")
-            return
-        }
-
-        navigationController?.pushViewController(toVC, animated: true)
-    }
-
```

Comment:
I wonder why this is not needed anymore?

Reply:
I forgot to make a note on this, this is dead code 😅 I was checking where `GetStartedViewController` is initialized in WPAuthenticator, and thought it's good to remove any unused code.